### DISCUSSION TITLE
aptos prover - coin->FA migration

### DISCUSSION THREAD

**Discussion by MoonShiesty** [aptos-labs/aptos-developer-discussions#337] [Questions] [answered]
### Discord user ID

moonshiesty

### Describe your question in detail.

with the current state of coin.spec.move it seems difficult to verify much about the balance of coin withdraw/deposit. is there an ETA on proving the coin->FA migration code?

in the meantime, is there anything that can be verified about coin balances or is the best thing to do just leave coin balances unverified for now?

 ```  
    /// Account is not frozen and sufficient balance.
    spec withdraw<CoinType>(
    account: &signer,
    amount: u64,
    ): Coin<CoinType> {
        // TODO(fa_migration)
    ```

**Comment by rahxephon89** [comment, marked as answer]
Thanks for the question, @MoonShiesty. We don't have a specific ETA for coin-->FA migration but will let you know once there is one.
